### ISSUE TITLE
Indicateur qualité : Renommer "Fraicheur" par "Péremption"

### ISSUE THREAD

**Issue by cyrilmorin** [etalab/transport-site#4047]
Depuis le début le terme de "fraicheur" pour l'indicateur qui permet de suivre la validité calendaire des données TC ne me semble pas approprié et surtout pas parlant.

=> Renommer partout le terme "Indicateur de fraicheur" par "Indicateur de péremption"

NB : même si cet indicateur évolue pour prendre en compte la profondeur de calendrier des données, le principe reste le même => on regarde la date de péremption pour calculer cet indicateur

**Comment by AntoineAugusti** [comment]
@cyrilmorin Pour information, l'indicateur `freshness` concerne aussi les ressources GTFS-RT et GBFS https://github.com/etalab/transport-site/issues/3590#issuecomment-1820897478

Cette information change-t-elle quelque chose pour toi ?

Faut-il modifier uniquement le libellé grand public ou également tout le code/ce qui est stocké en base de données et les exports ?

**Comment by vdegove** [comment]
Petite note : je me mets dans la tête du dev qui a inventé ce termes (je suppose @fchabouis), j’imagine que la raison c’est simplement qu’il privilégie un indicateur où « plus c’est haut, mieux c’est » pour être cohérent avec les autres.

Si on fait un indicateur de péremption, ça fait inverser l’indicateur : 1 c’est que le JDD n’est plus du tout à jour (péremption au max), 0 c’est que au contraire le JDD est au top.

Ou alors, on garde la notation actuelle sans l’inverser, mais du coup on un nom qui est inverse à la notation : on aurait un indicateur de péremption ou le 100% indique que la péremption est au minimum :upside_down_face:

**Comment by AntoineAugusti** [comment]
La PR initiale pour ce score https://github.com/etalab/transport-site/pull/3177

**Comment by cyrilmorin** [comment]
Hello, cette question de sémantique avait déjà été soulevée lorsque nous avions réalisé ce score avec Francis :)
La raison principale de ce terme était qu'on aurait aimé également y incorporer (après avoir expérimenté cela) une notion de profondeur des données et de nombre de mises à jour par an pour compléter cette 1ère vision et préciser la qualité et la confiance qu'on peut en avoir en tant que réutilisateur.

Je comprends tes questions @vdegove pour nous intra-PAN, mais ce terme n'est pas un terme utilisé dans l'éco-système et il est donc par conséquent mal compris.

Le terme de péremption qui est expliqué dans la [doc  ](https://doc.transport.data.gouv.fr/outils/outils-disponibles-sur-le-pan/indicateurs-de-qualite)) permet de comprendre le "concept" comme le dit Antoine avec 0%=pas bien vs 100%=bien comme les autres indicateurs.

Nous pourrions l'appeler score de "Non-péremption" également pour faire consensus !

Je valide avec les @etalab/transport-bizdev et vous confirme les termes validé à l'unanimité des suffrages ;)

**Comment by Brewennn** [comment]
Je suis d'accord pour ne pas mettre le terme "indicateur de péremption" pour les raisons évoquées par Vincent, surtout qu'avoir un indicateur inversée des autres rend la lecture plus compliquée et fait ressortir un aspect négatif de la publication. 

Je ne suis pas pour le terme "non-péremption". 

Le terme "fraîcheur" reste plus adapté mais je souhaiterais proposé le terme "indicateur de mise à jour" qui me parait mieux et plus parlant pour tous à mon avis (désolé j'en rajoute une couche :sweat_smile:)

**Comment by AntoineAugusti** [comment]
@cyrilmorin Ce ticket est-il encore pertinent ?

**Comment by cyrilmorin** [comment]
@AntoineAugusti je ferme
